tests/id_test.go

```diff
@@ -0,0 +1,44 @@
+package tests
+
+import (
+	"testing"
+
+	"github.com/ProtonMail/gluon/internal/response"
+
+	"github.com/ProtonMail/gluon/imap"
+	"github.com/stretchr/testify/require"
+)
+
+func TestIdNilNoData(t *testing.T) {
+	runOneToOneTest(t, "user", "pass", "/", func(c *testConnection, s *testSession) {
+		idResponse := response.IdResponse{ID: imap.NewIMAPIDFromVersionInfo(&TestServerVersionInfo)}
```

Comment:
Why did you decide to make this response type not use a constructor function thing like the others?